tools/releaseBuild/setVstsVariables.ps1

```diff
@@ -0,0 +1,11 @@
+$vstsVariables = @{
+    PSES_BRANCH = '2.0.0'
```

Comment:
why is this set to 2.0.0 vs 1.8.X?

Reply:
Because I cherry picked it from a branch building against `2.0.0`. Have changed it to `master`